modules/s3/locals.tf

```
@@ -2,9 +2,12 @@ locals {
   # The name of the bucket
   bucket_name = "devops-ic-assets-bucket"
 
+  # The name of ALB log bucket
+  alb_log_bucket_name = "devops-ic-alb-log-bucket"
```

The names start become confusing, let's make them clearer, e.g
```suggestion
  asset_bucket_name = "devops-ic-assets-bucket"

  # The name of ALB log bucket
  alb_log_bucket_name = "devops-ic-alb-log-bucket"
```